Make wp8-prod fail: Cannot read property 'replace' of undefined

Make fails for wp8-prod under msys. Windows build instructions are sparse and I very well may not be doing this right, please advise if so.


	derrick@Derrick-Home /d/Dev/copay
	$ make wp8-prod
	cordova/build.sh WP8 --clear
	* Checking dependencies...
	Build directory is /d/Dev/copay/cordova
	Project directory is /d/Dev/copay/cordova/project
	* Creating project...
	Creating a new cordova project.
	* Adding WP8 platform...
	Adding wp8 project...
	Running command: cmd "/s /c "C:\Users\derrick.ALIENSEED\.cordova\lib\npm_cache\cordova-wp8\3.8.2\package\bin\create.bat d:\Dev\copay\cordova\project\platforms\wp8 com.bitpay.copay Copay""
	Creating Cordova Windows Project:
			App Name  : Copay
			Namespace : com.bitpay.copay
			Path      : d:\Dev\copay\cordova\project\platforms\wp8
	Copying necessary files to d:\Dev\copay\cordova\project\platforms\wp8
	Updating project files
	Discovered plugin "cordova-plugin-whitelist" in config.xml. Installing to the project
	Fetching plugin "cordova-plugin-whitelist@1" via npm
	Installing "cordova-plugin-whitelist" for wp8

	This plugin is only applicable for versions of cordova-android greater than 4.0. If you have a previous platform version, you do *not* need this plugin since the whitelist will be built in.

	* Installing plugins...
	Fetching plugin "https://github.com/florentvaldelievre/virtualartifacts-webIntent.git" via git clone
	Repository "https://github.com/florentvaldelievre/virtualartifacts-webIntent.git" checked out to git ref "master".
	Installing "com.virtualartifacts.webintent" for wp8
	* Using plugin phonegap-plugin-barcodescanner for Windows Phone 8
	Fetching plugin "https://github.com/phonegap/phonegap-plugin-barcodescanner.git" via git clone
	Repository "https://github.com/phonegap/phonegap-plugin-barcodescanner.git" checked out to git ref "master".
	Installing "phonegap-plugin-barcodescanner" for wp8
	Fetching plugin "cordova-plugin-splashscreen" via npm
	Installing "cordova-plugin-splashscreen" for wp8
	Fetching plugin "cordova-plugin-statusbar" via npm
	Installing "cordova-plugin-statusbar" for wp8
	Fetching plugin "https://github.com/EddyVerbruggen/LaunchMyApp-PhoneGap-Plugin.git" via git clone
	Repository "https://github.com/EddyVerbruggen/LaunchMyApp-PhoneGap-Plugin.git" checked out to git ref "master".
	Installing "cordova-plugin-customurlscheme" for wp8
	Error: Cannot read property 'replace' of undefined
	* ERROR. Exiting...
	make: *** [wp8-prod] Error 1

	derrick@Derrick-Home /d/Dev/copay
	$

We are getting the same error in plugin `customurlscheme`:

```
Installing "cordova-plugin-customurlscheme" for wp8
TypeError: Cannot call method 'replace' of undefined
    at C:\cygwin64\home\bitpay\copay\cordova\project\plugins\cordova-plugin-customurlscheme\src\wp8\hooks\add-uri-mapper.js:13:21
    at C:\Users\bitpay\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\node_modules\npm\node_modules\graceful-fs\graceful-fs.js:76:16
    at fs.js:208:20
    at C:\Users\bitpay\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\node_modules\npm\node_modules\graceful-fs\graceful-fs.js:230:16
    at Object.oncomplete (fs.js:108:15)
* ERROR. Exiting...
Makefile:35: recipe for target 'wp8-prod' failed
make: *** [wp8-prod] Error 1
```

We are working on a fix. Thanks for report

--- a/cordova/build.sh
+++ b/cordova/build.sh
@@ -103,7 +103,7 @@ if [ ! -d $PROJECT ]; then
   cordova plugin add cordova-plugin-statusbar
   checkOK
 
-  cordova plugin add https://github.com/EddyVerbruggen/LaunchMyApp-PhoneGap-Plugin.git --variable URL_SCHEME=bitcoin
+  cordova plugin add cordova-plugin-customurlscheme --variable URL_SCHEME=bitcoin
   checkOK
 
   cordova plugin add cordova-plugin-inappbrowser

